scripts/src/main/resources/scripts/command/gh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+GH_HOME="${DEVON_IDE_HOME}/software/gh"
+
+function doSetup(){
+  # Install GitHub CLI in the software folder
+
+  if [ ! -d "${GH_HOME}" ]
+  then
+    # Get leatest release
+    if [ -z "${GH_VERSION}" ] || [ "${1}" != "silent" ]
+    then
+      doEcho "Getting latest release..."
+      local GH_VERSION
+      GH_VERSION=$( curl -LsH "Accept: application/vnd.github.v3+json" https://api.github.com/repos/cli/cli/releases/latest | awk -F':' '/tag_name/ {print $2}' | awk -F'"' '{print $2}' | tr -d 'v' )
+
+      doEcho "Installing GitHub CLI..."
+      doInstall "-" "${GH_HOME}" "gh" "${GH_VERSION}"
+    fi
+  fi
+}
+
+function doRun(){
+  doSetup silent
+  doRunCommand "gh ${*}"
```

Comment:
Does this work? I think you should add the path like `doRunCommand "${HELM_HOME}/helm" "${@}"` to call the binary.